client/trino-client/src/main/java/io/trino/client/ClientTypeSignature.java

```diff
@@ -94,6 +96,19 @@ public String toString()
         if (arguments.isEmpty()) {
             return rawType;
         }
+
+        if (rawType.equals(TIME_WITH_TIME_ZONE)) {
+            return "time" + arguments.stream()
+                    .map(ClientTypeSignatureParameter::toString)
+                    .collect(joining(",", "(", ")")) + " with time zone";
```

Comment:
For this type, `arguments` is guaranteed to have a single element, so this could be just:


```suggestion
            return "time(" + arguments.get(0) + ") with time zone";
```

Reply:
Thanks, applied.